src/components/Todo/Todo.js

```diff
@@ -1,8 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
+import Input from 'components/Input/Input'
+import Button from 'components/Button/Button'
+import { FaEdit } from 'react-icons/fa'
+import { MdCancel } from 'react-icons/md'
 
-const Todo = ({ children, ...props }) => {
-  return <ListItem {...props}>{children}</ListItem>
+const Todo = ({ onUpdateText, id, value, children, ...props }) => {
```

Comment:
Por mera cuestión de orden y convención, los props se ordenan de la siguiente forma. `value` no es necesario (ver comentario más abajo)

- id
- props propios
- eventos
- props de react (children, style, className)

```suggestion
const Todo = ({ id, onUpdateText, children, ...props }) => {
```